### DISCUSSION TITLE
Configurable 'default' style theme

### DISCUSSION THREAD

**Discussion by dbauszus-glx** [GEOLYTIX/xyz#2163] [Ideas]
We would like an interface to configure the style.default for a layer.

The legend for the `basic` theme should show either a line / or area and an icon based on the default style.

With an editable flag it should be possible to edit.

The strokeColor, strokeWidth, strokeOpacity.

The fillColor, fillOpacity.

The icon.

It should be possible to select any of the svg_symbol types and define the parameter required by each of the symbols.

Select any svg_template loaded and defined the substitute parameter.

Provide an svg url or png.

Define the scale for the icon.

Additionally it should be possible to define an icon array.

**Comment by cityremade** [comment]
The style editor supporting edits on default feature style and storing user locale in review at this point. 
Icon would be a separate design task.
Detecting changes between workspace and user locales in the works also.

**Comment by RobAndrewHurst** [comment]
@dbauszus-glx @cityremade Can we close this?

**Comment by dbauszus-glx** [comment]
@RobAndrewHurst Please leave open for now. There is functionality in a plugin which I want to move into core.
